modules/nodes-descheduler-strategies.adoc

```diff
@@ -12,7 +12,7 @@ The `LowNodeUtilization` strategy finds nodes that are underutilized and evicts
 +
 The underutilization of nodes is determined by several configurable threshold parameters: CPU, memory, and number of Pods. If a node's usage is below the configured thresholds for all parameters (CPU, memory, and number of Pods), then the node is considered to be underutilized.
```

Comment:
@bergerhoffer Just want to double check that the same language doesn't need to be updated for _underutilization_ as well.

Reply:
I think it should be updated as well @lbarbeevargas .

@damemi what do you think ?

Reply:
Good question - @lbarbeevargas @kasturinarra according to the slack discussion that started this, all of the threshold parameters need to be met to qualify for underutilization, but only one of them needs to qualify for overutilization. @damemi can correct me if I'm wrong though

Reply:
This is correct (verified by [these comments in the descheduler code](https://github.com/kubernetes-sigs/descheduler/blob/master/pkg/descheduler/strategies/lownodeutilization.go#L392-L415)).